.swiftlint.yml

```diff
@@ -5,6 +5,7 @@ excluded:
   - Pods
   - Sources/MapboxMaps/Style/Generated
   - Tests/MapboxMapsTests/Style/Generated
+  - scripts/depsvalidator/.build
```

Comment:
can this `.build` also go in the `.gitignore`

Reply:
It already is: https://github.com/mapbox/mapbox-maps-ios/blob/ah/depsvalidator/.gitignore#L3

Reply:
nice yeah I saw that later